src/app/register/register.component.ts

```diff
@@ -91,8 +115,15 @@ export class RegisterComponent implements OnInit, OnDestroy {
     }
 
     this.processing = true;
-    this.captcha.reset();
-    this.captcha.execute();
+    if (! this.friendlyCaptchaSolution) {
```

Comment:
Don't we want to avoid reCAPTCHA running any time Friendly Captcha is enabled, not just when it already completed?

Reply:
Probably yes. I was imagining while we are prototyping Friendly Captcha we'd have both on the page and accept either solution, but I don't think its necessary.

Reply:
Changed in 396453a57